docs/changes.rst

```diff
@@ -54,6 +54,11 @@ Released: not yet
   a class is a subclass of a class. Both classes can be specified as classnames
   or as ``CIMClass`` objects.
 
+* Added support for translating the values of ``PUnit`` and ``Units``
+  qualifiers into human readable SI conformant unit strings, via new
+  functions ``pywbem.siunit_obj()`` and ``pywbem.siunit()``.
```

Comment:
Should we not that this only works on servers that support classes?

Reply:
These functions do not perform class operations under the covers. However, the code in pywbemtools that uses them does. I added a note that "these functions do not perform any class operations; they take the qualifiers as input."